action.php

```diff
@@ -282,7 +282,7 @@ function handle_secedit_button(Doku_Event &$event, $params) {
         } elseif ($data['target'] == 'plugin_include_end') {
             array_shift($page_stack);
         } elseif ($data['target'] == 'plugin_include_editbtn') {
-            if ($page_stack[0]['writable']) {
+            if (is_array($page_stack) && isset($page_stack[0]) && $page_stack[0]['writable']) {
```

Comment:
`if ($page_stack[0]['writable'] ?? false) {` might also work and be easier and shorter?

Reply:
Um, maybe not? Ref #298

Reply:
I don't get your objection. There is nothing in #298 that invalidates my suggestion AFAICT.

The only potential issue I can see is that this would require PHP version 7.0 or later. As 7.4. has reached end-of-life I'm not too concerned about that though.

But if you don't like to use the [Null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op) that's fine.

Reply:
Aha! OK. I'm *not* a PHP expert, in fact I don't know it at all. [cough]

Reply:
OK done - thank you!